<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionControll</class>
 <widget class="QWidget" name="MotionControll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QPushButton" name="btn_GoEndMannul">
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_GoBeginMannul">
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_Connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_Zero">
      <property name="text">
       <string>OriginalPoint</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_GoEnd">
      <property name="text">
       <string>GoEnd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_Stop">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Distance"/>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="CheckBox_DistanceSet">
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_RotateStart">
      <property name="text">
       <string>Rotate Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_Disconnect">
      <property name="text">
       <string>DisConnect</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_HorisonPos">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btn_RotateStop">
      <property name="text">
       <string>Rotate Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
